From e6fee6771553a2bb8f13da61890cf4b0f0d352c7 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Thu, 6 Aug 2020 14:51:51 +0800
Subject: [PATCH] ARM: dts: rv1109: remove the dts repeated with rv1126

Now the code has realized compatible with rv1109 and rv1126.
It is no longer need these duplicate dts of rv1109.
At the same time, the configuration of LP3 and DDR3 is the same.
Remove them together for reduces maintenance work.

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ie732509aa821d37376a725fc12c358915c7d6869
---
 arch/arm/boot/dts/Makefile                    |  5 -----
 arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts     | 17 -----------------
 .../boot/dts/rv1109-evb-ddr3-v12-tb-emmc.dts  | 19 -------------------
 arch/arm/boot/dts/rv1109-evb-ddr3-v12.dts     | 15 ---------------
 arch/arm/boot/dts/rv1109-evb-lp3-v10.dts      | 18 ------------------
 arch/arm/boot/dts/rv1126-evb-lp3-v10.dts      | 18 ------------------
 6 files changed, 92 deletions(-)
 delete mode 100644 arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
 delete mode 100644 arch/arm/boot/dts/rv1109-evb-ddr3-v12-tb-emmc.dts
 delete mode 100644 arch/arm/boot/dts/rv1109-evb-ddr3-v12.dts
 delete mode 100644 arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
 delete mode 100644 arch/arm/boot/dts/rv1126-evb-lp3-v10.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index e0d159ffe18a..4da2085ed052 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -844,11 +844,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += \
 	rv1108-evb.dtb \
 	rv1109-38-v10-spi-nand.dtb \
 	rv1109-ai-cam-ddr3-v1.dtb \
-	rv1109-evb-ddr3-v10.dtb \
-	rv1109-evb-ddr3-v12.dtb \
 	rv1109-evb-ddr3-v12-facial-gate.dtb \
-	rv1109-evb-ddr3-v12-tb-emmc.dtb \
-	rv1109-evb-lp3-v10.dtb \
 	rv1109-fpga.dtb \
 	rv1126-38x38-v10-spi-nor.dtb \
 	rv1126-ai-cam-ddr3-v1.dtb \
@@ -862,7 +858,6 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += \
 	rv1126-evb-ddr3-v12-tb-emmc.dtb \
 	rv1126-evb-ddr3-v12-tb-spi-nor.dtb \
 	rv1126-evb-ddr3-v13.dtb \
-	rv1126-evb-lp3-v10.dtb \
 	rv1126-iotest-v10.dtb \
 	rv1126-ipc2-ddr3-v10.dtb \
 	rv1126-sphericalipc-ddr3-v10.dtb \
diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
deleted file mode 100644
index 5c8c17c6976c..000000000000
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v10.dts
+++ /dev/null
@@ -1,17 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*
- * Copyright (c) 2019 Fuzhou Rockchip Electronics Co., Ltd.
- */
-
-/dts-v1/;
-#include "rv1109.dtsi"
-#include "rv11xx-evb-v10.dtsi"
-
-/ {
-	model = "Rockchip RV1109 EVB DDR3 Board";
-	compatible = "rockchip,rv1109-evb-ddr3-v10", "rockchip,rv1109";
-
-	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
-	};
-};
diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-tb-emmc.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v12-tb-emmc.dts
deleted file mode 100644
index 1e51b4cf2c7d..000000000000
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-tb-emmc.dts
+++ /dev/null
@@ -1,19 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*
- * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
- */
-
-/dts-v1/;
-#include "rv1109.dtsi"
-#include "rv11xx-evb-v12.dtsi"
-#include "rv11xx-thunder-boot-emmc.dtsi"
-#include "rv11xx-evb-thunder-boot.dtsi"
-
-/ {
-	model = "Rockchip RV1109 EVB DDR3 V12 with eMMC (Thunder Boot)";
-	compatible = "rockchip,rv1109-evb-ddr3-v12-tb-emmc", "rockchip,rv1109";
-
-	chosen {
-		bootargs = "loglevel=0 initcall_nr_threads=-1 initcall_debug=0 printk.devkmsg=on root=/dev/rd0 console=ttyFIQ0 snd_aloop.index=7 driver_async_probe=dwmmc_rockchip rk.root2nd=/dev/mmcblk0p7";
-	};
-};
diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v12.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v12.dts
deleted file mode 100644
index 3355d42e5440..000000000000
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v12.dts
+++ /dev/null
@@ -1,15 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*
- * Copyright (c) 2020 Rockchip Electronics Co., Ltd.
- */
-/dts-v1/;
-#include "rv1109.dtsi"
-#include "rv11xx-evb-v12.dtsi"
-/ {
-	model = "Rockchip RV1109 EVB DDR3 Board";
-	compatible = "rockchip,rv1109-evb-ddr3-v12", "rockchip,rv1109";
-
-	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
-	};
-};
diff --git a/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts b/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
deleted file mode 100644
index b556f7efc087..000000000000
--- a/arch/arm/boot/dts/rv1109-evb-lp3-v10.dts
+++ /dev/null
@@ -1,18 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*
- * Copyright (c) 2019 Fuzhou Rockchip Electronics Co., Ltd.
- */
-
-/dts-v1/;
-
-#include "rv1109.dtsi"
-#include "rv11xx-evb-v10.dtsi"
-
-/ {
-	model = "Rockchip RV1109 EVB LPDDR3 Board";
-	compatible = "rockchip,rv1109-evb-lp3-v10", "rockchip,rv1109";
-
-	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
-	};
-};
diff --git a/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts b/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts
deleted file mode 100644
index a8c721b9ea3d..000000000000
--- a/arch/arm/boot/dts/rv1126-evb-lp3-v10.dts
+++ /dev/null
@@ -1,18 +0,0 @@
-// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
-/*
- * Copyright (c) 2019 Fuzhou Rockchip Electronics Co., Ltd.
- */
-
-/dts-v1/;
-
-#include "rv1126.dtsi"
-#include "rv11xx-evb-v10.dtsi"
-
-/ {
-	model = "Rockchip RV1126 EVB LPDDR3 Board";
-	compatible = "rockchip,rv1126-evb-lp3-v10", "rockchip,rv1126";
-
-	chosen {
-		bootargs = "earlycon=uart8250,mmio32,0xff570000 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait snd_aloop.index=7";
-	};
-};
-- 
2.35.3

